keps/sig-auth/541-external-credential-providers/README.md

```diff
@@ -258,25 +258,49 @@ type ExecCredentialSpec struct {
 
 // Cluster contains information to allow an exec plugin to communicate with the
 // kubernetes cluster being authenticated to.
+//
+// To ensure that this struct contains everything someone would need to communicate
+// with a kubernetes cluster (just like they would via a kubeconfig), the fields
+// should shadow "k8s.io/client-go/tools/clientcmd/api/v1".Cluster, with the exception
+// of InsecureSkipVerify (in order to discourage its use) and CertificateAuthority
```

Comment:
Open question: is it OK to not include `InsecureSkipTLSVerify` here? Need to think through what guidance to exec plugins would be.